crypt.go

```diff
@@ -725,6 +740,24 @@ func InitializeLUKS2Container(devicePath, label string, key []byte, options *Ini
 				"--luks2-keyslots-size", fmt.Sprintf("%dk", options.KeyslotsAreaKiBSize))
 		}
 	}
+	if pbkdfOptions != nil {
+		if pbkdfOptions.PbkdfMmemory != 0 {
+			args = append(args,
+				"--pbkdf-memory", fmt.Sprintf("%d", pbkdfOptions.PbkdfMmemory))
+		}
+		if pbkdfOptions.PkdfForceIterations != 0 {
+			args = append(args,
+				"--pbkdf-force-iterations", fmt.Sprintf("%d", pbkdfOptions.PkdfForceIterations))
+		} else {
+			args = append(args, "--iter-time", "100")
```

Comment:
we should also make iter-itme itself an option (and use the defaults if it's unset as well)

Reply:
for this one maybe is not so important